server/bin/gen_fake_report.py

```diff
@@ -38,16 +61,19 @@ class VirtWhoReportGenerator(AbstractVirtWhoReportGenerator):
     def _generate(self):
         report = {}
         for x in range(0, self.hypervisors):
-            report[str(uuid.uuid4())] = [self.gen_guest() for y in
-                range(0, self.guests)]
+            report[str(uuid.uuid4())] = [
+                self.gen_guest() for y in range(0, self.guests)
+            ]
         return report
 
 
 class AsyncVirtWhoReportGenerator(AbstractVirtWhoReportGenerator):
 
     def gen_hypervisor(self, num_guests, min_guests=0, max_guests=None):
-        return {"uuid": str(uuid.uuid4()),
-                "guests": [self.gen_guest() for x in range(0, num_guests)]}
+        return {"hypervisorId": {"hypervisorId": str(uuid.uuid4())},
+                "name": "generated_hypervisor-" + gen_random_string(),
+                "guests": [self.gen_guest() for x in range(0, num_guests)],
```

Comment:
I don't know why this field was originally called `guests`, but it's wrong. When posting the generated report to the server, it ends up reporting the hypervisor but with 0 guests, because the field is called `guestIds`, not `guests`

Reply:
You're right, this was not working with ```guests```, changed it to ```guestIds```